examples/examples_readme.txt

```diff
@@ -0,0 +1,2 @@
+This folder contains example PROTEUS runs. They are dated to when they were 
```

Comment:
Ok for now, for the next example plots and configs I would suggest to also add the exact PROTEUS version with which they were generated. In this case, I presume this was commit `a1d24fe`.

Reply:
I think that's a good idea. These were created with commit [43a98a1](https://github.com/FormingWorlds/PROTEUS/pull/47/commits/43a98a113ced8a14acbbc24370b80a180da68a02), which differs from the merged version only by the ReadTheDocs changes.